### ISSUE TITLE
button to set imaMask to transparent

### ISSUE THREAD

**Issue by MSchnei** [ofgulban/segmentator#28]
on/off switch

**Comment by MSchnei** [comment]
Button with this functionality is now added.
Position of button is up for discussion.

![imamaskbutton](https://cloud.githubusercontent.com/assets/15090072/18795038/d72b3384-81c2-11e6-98fa-31d34e1314f9.gif)

**Comment by ofgulban** [comment]
Well nothing can look worse than the current radio buttons :D, I think the position of the transparency toggle button is fine.

**Comment by MSchnei** [comment]
I also added a slider. This gives much more flexibility. Still i left the button to toggle quickly between high and low transparency. I also added this feature for segmentator main.
@ofgulban: Let me know what you think!

![screenshot from 2016-09-26 19 12 20](https://cloud.githubusercontent.com/assets/15090072/18844445/8b33d5dc-841d-11e6-8729-a3a54d6fcc0e.png)
![screenshot from 2016-09-26 19 12 19](https://cloud.githubusercontent.com/assets/15090072/18844443/8b31c1a2-841d-11e6-9226-5597a18c55c2.png)
![screenshot from 2016-09-26 19 12 18](https://cloud.githubusercontent.com/assets/15090072/18844444/8b32f036-841d-11e6-8d23-0c474dde3a2c.png)
